### DISCUSSION TITLE
How to resolve OrganizationUnit Filter

### DISCUSSION THREAD

**Discussion by sgalcheung** [abpframework/abp#7967] [General] [answered]
I have a requirement to filter data through organization structure, When I refer to the implementation of multi-tenancy, CurrentUser does not hava an organizationUnitId, and then I found that the IdentityUserOrganzationUnit has a user-organization relationship, so should I create a CurrentUserOrganizationUnit? Is there a recommended way to implement id?
The main problem is the relationship between users and organizations.

**Comment by realLiangshiwei** [comment, marked as answer]
See https://github.com/abpframework/abp/issues/4350

**Reply by sgalcheung** [reply]
The question is how to get the orgainzation?

**Reply by sgalcheung** [reply]
The [#4350 ](https://github.com/abpframework/abp/issues/4350)caused me some misunderstandings, you can use, or use?
